### PR TITLE
Update codeowners to individual handles to prepare for move

### PR DESCRIPTION
**Pull request by annafil**
# Description

Preparing to move this repo into the /apache github org https://issues.apache.org/jira/browse/INFRA-26049 

We will lose our GitHub team from the Polaris-Catalog org, so updating code owners with individual humans. 

# Checklist:

Please delete options that are not relevant.

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If adding new functionality, I have discussed my implementation with the community using the linked GitHub issue
- [ ] I have signed and submitted the [ICLA](https://github.com/polaris-catalog/polaris/blob/main/ICLA.md) and if needed, the [CCLA](https://github.com/polaris-catalog/polaris/blob/main/CCLA.md). See [Contributing](https://github.com/polaris-catalog/polaris/blob/main/CONTRIBUTING.md) for details.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 
-* @polaris-catalog/polaris
+* @jbonofre @anoopj @RussellSpitzer @adutra @laurentgo @snazy @hivivo @flyrain @dennishuo @takidau @aihuaxu @jackye1995 @collado-mike @dimas-b
```
Comment (jbonofre) [inline .github/CODEOWNERS:17]: In preparation for ASF move, we should have only the PPMC (listed in the proposal). It means `@jbonofre @ashvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995`

See: https://cwiki.apache.org/confluence/display/INCUBATOR/PolarisProposal

Comment (annafil) [reply .github/CODEOWNERS:17]: Roger that -- thank you for the clarification! Will update

Comment (sfc-gh-mcollado) [reply .github/CODEOWNERS]: I think it'll be important to have me and/or @dennishuo in here, at least very soon after the move. I think we have the deepest knowledge of the code base and ought to be involved with reviewing most of the changes to the actual business logic

Comment (collado-mike) [reply .github/CODEOWNERS]: I think it'll be important to have me and/or @dennishuo in here, at least very soon after the move. I think we have the deepest knowledge of the code base and ought to be involved with reviewing most of the changes to the actual business logic

Comment (annafil) [reply .github/CODEOWNERS]: Resolved this offline -- we will add committers publicly based on ASF processes like https://community.apache.org/contributors/becomingacommitter.html

**Comment by jbonofre** [comment]
@annafil GitHub says that `CODEOWNERS` contains error. Do you have the same message in your GitHub ?

**Comment by flyrain** [comment]
I can see the same error message, but the format looks look to me. Here is the doc about the code owner format.

**Comment by flyrain** [comment]
```
Unknown owner on line 17: make sure @jbonofre exists and has write access to the repository
* @jbonofre @ashvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
Unknown owner on line 17: make sure @ashvina exists and has write access to the repository
* @jbonofre @ashvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
Unknown owner on line 17: make sure @anoopj exists and has write access to the repository
* @jbonofre @ashvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
Unknown owner on line 17: make sure @RussellSpitzer exists and has write access to the repository
* @jbonofre @ashvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
Unknown owner on line 17: make sure @snazy exists and has write access to the repository
…hvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
Unknown owner on line 17: make sure @vvcephei exists and has write access to the repository
…anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
Unknown owner on line 17: make sure @hivivo exists and has write access to the repository
…ssellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995
```
Could be a hidden char or something, maybe we just merge and see.

**Comment by annafil** [comment]
I think this is happening because GitHub checks that these folks have permissions at the repo level. @jbonofre how long does the LDAP stuff take to propagate? To resolve this error, each of the folks on this codeowners file needs to be provisioned write access on the repo (either directly as a user, or via whatever team structure ASF users internally in their LDAP)

**Comment by jbonofre** [comment]
@annafil it should be done already. Let me check.

**Comment by annafil** [comment]
It might also help to re-run the checks on the last commit, it might trigger checking the codeowners file for validity

**Review by jbonofre — APPROVED** [review]

**Comment by jbonofre** [comment]
I worked with infra, PPMC should all have permission now to review/merge/push.

**Pull request merged**